webots_ros2/CHANGELOG.rst

```diff
@@ -2,6 +2,24 @@
 Changelog for package webots_ros2
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2022.1.5 (2022-11-17)
+------------------
+* Update the calculation of CameraRecognitionObject messages to the RDF convention of R2022b.
+
+2022.1.4 (2022-XX-XX)
+------------------
+* Fix the camera focal length in the CameraInfo topic.
+* Upgraded to urdf2webots 2.0.3
```

Comment:
2022.1.4 hasn't been released yet, entries under 2022.1.5 should still be put under 2022.1.4